playground/tests/test_alive.py

```diff
@@ -0,0 +1,13 @@
+import pytest_container
+from .conftest import TW
+import time
+
+CONTAINER_IMAGES = [TW]
+
+
+def test_etc_os_release_present(auto_container: pytest_container.container.ContainerData):
+    assert auto_container.connection.file("/etc/os-release").exists
+
+def test_alive(auto_container: pytest_container.container.ContainerData, host):
+    time.sleep(10)
```

Comment:
This will give you a pretty hefty delay on each test. You can try something similar to this: https://github.com/SUSE/BCI-tests/blob/aa25debbe104f5c856edd54cd1d653a75bb4b60e/tests/test_tomcat.py#L36

Reply:
Thanks. That is a good idea.